clients/cpp-console/include/WindowsAudioPlayer.h

```diff
@@ -61,51 +60,35 @@ namespace AudioPlayer
             ~WindowsAudioPlayer();

```

Comment:
Do we have to document the WindwosAudioPlayer() and LinuxAudioPlayer()? it's the same text we have in the IAudioPlayer. We should avoid duplicating documentation comments. Perhaps the Linux and Windows implementation headers just say "see IAudioPlayer for more details". If there is something unique to that platform, the comments should mention it, but otherwise IAudioPlayer should have the generic comments.

Reply:
I'm fine with that